.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+Applicable spec: <link>
+
+### Overview
+
+<!-- A high level overview of the change -->
+
+### Rationale
+
+<!-- The reason the change is needed -->
+
+### Juju Events Changes
+
+<!-- Any changes to the juju events being observed (newly added, significantly modified or deleted) -->
+
+### Module Changes
+
+<!-- Any high level changes to modules and why (Service, Observer, helper) -->
+
+### Library Changes
+
+<!-- Any changes to charm libraries -->
+
+### Checklist
+
+- [ ] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
+- [ ] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
+- [ ] The changes are compliant with [ISD054 - Manging Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
```

Comment:
Should we add something about that documentation is updated here?

Reply:
Added a few more items
```diff
+ - [ ] The documentation is generated using `src-docs`
+ - [ ] The documentation for charmhub is updated.
+ - [ ] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)
```